### DISCUSSION TITLE
Notifications for failed PRs include cancelled runs

### DISCUSSION THREAD

**Discussion by lennertcc** [community/community#145786] [Mobile] [answered]
I enabled notifications workflows, including the option "only failed runs". Apparently that also includes cancelled runs, which I don't think is fitting the description and also not my expectations.

My flow is that often times a new commit comes in on the PR, cancelling a running action and starting a new one. I'm not at all interested in this event.

Please only send notifications for failed workflows, or make the setting more granular.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.187.0 (147865513)
Device: iPhone SE (3rd generation) (iOS 18.1.1)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @lennertcc thanks for writing in. I have passed this onto the team for consideration!
